dashboard/src/components/Header/Menu.scss

```diff
@@ -67,6 +67,7 @@
 
 .dropdown-divider {
   opacity: 0.2;
+  border-bottom-color: var(--cds-global-color-gray-50, #fafafa) !important;
```

Comment:
avoid `!important` :)

Reply:
The point is that the dark.mode clr-ui is loaded by helmet directly as the last element (ie, after the css bundle). So the selector from the `.dropdown-divider` selector from clr-ui-dark has more priority.

We could add an id and apply the style there, but I wanted this PR to be css-only. Would you prefer adding the id selector?

Reply:
you just need to make the selector more scoped, if you set it as `div .dropdown-menu .dropdown-divider` for example, it should work.

Reply:
Yep, you're totally right, it works. Thanks!! :)